src/adapters/mqtt/index.ts

```diff
@@ -127,7 +129,7 @@ class MqttAdapter extends Adapter {
 
   private subscribe(channels: string[]) {
     channels.forEach((channel) => {
-      const operation = this.parsedAsyncAPI.channel(channel).publish()
+      const operation = this.parsedAsyncAPI.channels().get(channel).json().subscribe()
```

Comment:
Nope. Never access json properties like this. You're breaking the whole purpose of the new parser. Also, this won't work with v3 because there's no publish or subscribe in the new version. Instead, use the equivalent methods. Here, you should try to find the "send" operation that is associated with the channel `channel`. You have to iterate `this.parsedAsyncAPI.operations()`.